integration-tests/src/server.js

```diff
@@ -76,6 +77,7 @@ const registerTestEndpoint = ({ fn, name, timeout = defaultTimeout }) => {
 }
 
 _.forEach(registerTestEndpoint, [
+  testAnalysisContextBar,
   testCreateInteractiveAnalysis,
   testFindWorkflow,
```

Comment:
Interesting… I haven't registered either of my new tests here, but they run fine in ci. @kyuksel do you know what this file is used for?

Reply:
I think this is this alternate way of running the tests through Bueller, that nobody actually does…. I wonder if it's time to delete this file this stuff. I missed the last day of the training, when perhaps Brett talked about this part of the test infrastructure.

Reply:
I think so but I'm not sure either. @breilly2 do you know?

As for the possibility of deprecating Bueller, I think it'll be good to discuss that with the QA team since they'll take over the ownership of the testing infrastructure. Until then I'd suggest we keep the list up to date.

Reply:
I am not inclined to try to keep the list up-to-date. If this infrastructure becomes useful at some point, all the missing tests could be added at that time (there are a number of them). I think it is more likely that we will ultimately delete this capability.

Reply:
I'm fine with that.